src/logion/migration/1636720219315-InitialSchema.ts

```diff
@@ -24,7 +24,11 @@ export class InitialSchema1636720219315 implements MigrationInterface {
                                          "phone_number"      character varying(255),
                                          CONSTRAINT "PK_loc_request" PRIMARY KEY ("id")
                                      )`);
+        } else {
+            await queryRunner.query(`ALTER TABLE "loc_request"
+                RENAME CONSTRAINT "PK_6e5754f2cc6563e70656d4b577f" TO "PK_loc_request"`);
```

Comment:
Did you check that these auto-generated constraint names are the same on MVP hosts and DEV?

Reply:
Yes. Those generated names are the same on node01, node02 and node03, dev01, dev02, dev03 and my local database if I generate the schema using the previous option `synchronize: true`. 

For the record here is a query that helped me to check:
```sql
SELECT rel.relname, con.conname
FROM pg_catalog.pg_constraint con
         INNER JOIN pg_catalog.pg_class rel
                    ON rel.oid = con.conrelid
         INNER JOIN pg_catalog.pg_namespace nsp
                    ON nsp.oid = connamespace
WHERE nsp.nspname = 'public'
AND rel.relname <> 'migrations'
ORDER BY rel.relname, con.conname;
```